config/content.json

```diff
@@ -8,6 +8,34 @@
             "/docs/cloud-images/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-images",
             "/docs/cloud-load-balancers/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-load-balancers"
         },
+        "/docs/cloud-block-storage/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-block-storage"
+        },
+        "/docs/cloud-dns/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-dns"
+        },
+        "/docs/cloud-cdn/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-cdn"
+        },
+        "/docs/cloud-databases/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-databases"
+        },
+        "/docs/cloud-backup/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-backup/v1"
+        },
+        "/docs/cloud-orchestration/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-orchestration"
+        },
+        "/docs/rackconnect/v3/developer-guide/": "https://github.com/rackerlabs/docs-cloud-rackconnect"
+        },
+        "/docs/cloud-queues/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-queues"
+        },
+        "/docs/cloud-networks/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-networks"
+        },
+        "/docs/cloud-big-data/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-big-data/v2"
+        },
+        "/docs/cloud-autoscale/v1/developer-guide/": "https://github.com/rackerlabs/otter"
+        },
+        "/docs/cloud-servers/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-servers"
+        },
+        "/docs/cloud-files/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-files"
+        },
+        "/docs/cloud-metrics/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-databases"
+        },
```

Comment:
1. This isn't valid JSON; there are a bunch of extra `},` lines and no trailing comma for each element.
2. I've standardized on trailing slashes on the content ID prefixes. Deconst used to normalize them but I don't think it does right now, so you probably _need_ them here for it to work.